geometry-data-service.md

```diff
@@ -0,0 +1,94 @@
+# Geometry Data Service
+
+The Geometry Data Service (GDS) is a lightweight API that has been developed in Loc-I to facilitate 
+* minting URIs for geometries to enable Linked Data functionality
+* content-negotiation for formats and representations of the geometry
+* search by point for relevant geometries across the geometry store
+* parameterise the profile of the geometry representation for a Loc-I Feature (geometry of a feature with a resolution in metres, bounding box, centroid, simplified vs. complex geom, projection/CRS)
+
+![Geometry Data Service architecture](images/geometry-data-service.png "Geometry Data Service architecture")
+
```

Comment:
I wonder if we could put a GDS API over DGGS service?

Reply:
We probably could... do you mean over a DGGS engine? GA have an example here:
http://ec2-52-63-73-113.ap-southeast-2.compute.amazonaws.com/AusPIX-DGGS-dataset/ausPIX/R78523 

I think we could explore - it would enable interfaces to be standardised, though we'd need to think about how we interface a DGGS engine vs. Postgres DB.